### PR TITLE
[bugfix] skip tests of scatter & gather

### PR DESCRIPTION
**Pull request by StrongSpoon**
### PR Category

CI/CD
### Type of Change

Bug Fix
### Description

skip tests of scatter and gather
remove benchmark of gelu_tanh and gelu_tanh_bwd
rename some other benchmarks
### Progress

- [ ] Change is properly reviewed (1 reviewer required, 2 recommended).
- [ ] Change is responded to an issue.
- [ ] Change is fully covered by a UT.

### CODE REVIEW

**Review by Bowen12992 — APPROVED** [review]
LGTM

File: tests/test_reduction_ops.py [hunk 1]
```diff
@@ -704,6 +704,7 @@ def test_accuracy_vectornorm(shape, ord, dim, keepdim, dtype):
     gems_assert_close(res_out, ref_out, dtype)
 
 
+@pytest.mark.skipif(True, reason="operator undone")
```
Comment (iclementine) [inline tests/test_reduction_ops.py]: Just use skip, since no real conditions now.

**Review by iclementine — DISMISSED** [review]

File: tests/test_reduction_ops.py (hunk 1 above)
Comment (StrongSpoon) [reply tests/test_reduction_ops.py]: done

**Review by Bowen12992 — APPROVED** [review]
LGTM

**Pull request merged**